lib/repl.js

```diff
@@ -851,14 +851,15 @@ function complete(line, callback) {
           // Exclude versioned names that 'npm' installs.
           continue;
         }
-        if (exts.indexOf(ext) !== -1) {
+        abs = path.resolve(dir, name);
+        isDirectory = fs.statSync(abs).isDirectory();
```

Comment:
Looking at this again, I'm wondering if this should be moved back into a try-catch to avoid creating an error if the path doesn't exist.